include/mg_procedure.h

```diff
@@ -33,20 +38,12 @@ extern "C" {
 
 /// All functions return an error code that can be used to figure out whether the API call was successful or not. In
 /// case of failure, the specific error code can be used to identify the reason of the failure.
-enum MGP_NODISCARD mgp_error {
-  MGP_ERROR_NO_ERROR = 0,
-  MGP_ERROR_UNKNOWN_ERROR,
-  MGP_ERROR_UNABLE_TO_ALLOCATE,
-  MGP_ERROR_INSUFFICIENT_BUFFER,
-  MGP_ERROR_OUT_OF_RANGE,
-  MGP_ERROR_LOGIC_ERROR,
-  MGP_ERROR_DELETED_OBJECT,
-  MGP_ERROR_INVALID_ARGUMENT,
-  MGP_ERROR_KEY_ALREADY_EXISTS,
-  MGP_ERROR_IMMUTABLE_OBJECT,
-  MGP_ERROR_VALUE_CONVERSION,
-  MGP_ERROR_SERIALIZATION_ERROR,
-};
+MGP_ENUM_CLASS_TAG MGP_NODISCARD mgp_error{MGP_ERROR_NO_ERROR,           MGP_ERROR_UNKNOWN_ERROR,
+                                           MGP_ERROR_UNABLE_TO_ALLOCATE, MGP_ERROR_INSUFFICIENT_BUFFER,
+                                           MGP_ERROR_OUT_OF_RANGE,       MGP_ERROR_LOGIC_ERROR,
+                                           MGP_ERROR_DELETED_OBJECT,     MGP_ERROR_INVALID_ARGUMENT,
+                                           MGP_ERROR_KEY_ALREADY_EXISTS, MGP_ERROR_IMMUTABLE_OBJECT,
+                                           MGP_ERROR_VALUE_CONVERSION,   MGP_ERROR_SERIALIZATION_ERROR};
```

Comment:
```suggestion
                                           MGP_ERROR_VALUE_CONVERSION,   MGP_ERROR_SERIALIZATION_ERROR,};
```
If you put an "extra" command at the end of the listings, `clang-format` will put every value in a new line. It is not mandatory, but for such lists I think it is more readable.

Reply:
Adding an extra ',' changes the layout but keeps two values per line so that does not solve the problem:
```
MGP_ENUM_CLASS MGP_NODISCARD mgp_error{
    MGP_ERROR_NO_ERROR,           MGP_ERROR_UNKNOWN_ERROR,
    MGP_ERROR_UNABLE_TO_ALLOCATE, MGP_ERROR_INSUFFICIENT_BUFFER,
    MGP_ERROR_OUT_OF_RANGE,       MGP_ERROR_LOGIC_ERROR,
    MGP_ERROR_DELETED_OBJECT,     MGP_ERROR_INVALID_ARGUMENT,
    MGP_ERROR_KEY_ALREADY_EXISTS, MGP_ERROR_IMMUTABLE_OBJECT,
    MGP_ERROR_VALUE_CONVERSION,   MGP_ERROR_SERIALIZATION_ERROR,
};
```
Alternatively I can disable clang format on this portion only as:
```
// clang-format off
MGP_ENUM_CLASS MGP_NODISCARD mgp_error{
    MGP_ERROR_NO_ERROR,
    MGP_ERROR_UNKNOWN_ERROR,
    MGP_ERROR_UNABLE_TO_ALLOCATE,
    MGP_ERROR_INSUFFICIENT_BUFFER,
    MGP_ERROR_OUT_OF_RANGE,
    MGP_ERROR_LOGIC_ERROR,
    MGP_ERROR_DELETED_OBJECT,
    MGP_ERROR_INVALID_ARGUMENT,
    MGP_ERROR_KEY_ALREADY_EXISTS,
    MGP_ERROR_IMMUTABLE_OBJECT,
    MGP_ERROR_VALUE_CONVERSION,
    MGP_ERROR_SERIALIZATION_ERROR,
};
// clang-format on
```
What's your call on that?

Reply:
Oh, then I wasn't correct, sorry for that. Having an automatically formatted,but a bit off format is better in my opinion than a good, but manually formatted code. So I would say let's go with the weird format with the ending comma, so at least the braces look good.